src/DataWrangling/ECCO2.jl

```diff
@@ -1,19 +1,50 @@
 module ECCO2
 
+export ECCO2Data, ecco2_field, ecco2_center_mask, adjusted_ecco_tracers, initialize!
+
+using ClimaOcean.DataWrangling: fill_missing_values!
+using ClimaOcean.InitialConditions: three_dimensional_regrid!
+
 using Oceananigans
+using Oceananigans: architecture
 using Oceananigans.BoundaryConditions
+using Oceananigans.Utils
+using KernelAbstractions: @kernel, @index
 using NCDatasets
 
-temperature_filename = "THETA.1440x720x50.19920102.nc"
-salinity_filename = "SALT.1440x720x50.19920102.nc"
-effective_ice_thickness_filename = "SIheff.1440x720.19920102.nc"
+import Oceananigans.Fields: set!
+
+# Ecco field used to set model's initial conditions
+struct ECCO2Data
```

Comment:
```suggestion
struct ECCO2Metadata
```

this isn't the data itself, but not sure what to call it